core/data_access/base_repositories/repository.py

```diff
@@ -21,6 +20,10 @@ def create(self, obj: Dict[str, Any]) -> T:
         instance = self._model(**obj)
         instance.save()
         return instance
+    
+    def create_2(self, obj: T) -> T:
```

Comment:
why not just overwrite the existing create?

Reply:
Forgot. Will Refactor